docs/reference/cloud/list-of-supported-clouds/the-amazon-ec2-cloud-and-juju.md

```diff
@@ -67,34 +67,32 @@ Example: `vpc-a1b2c3d4`
-| - {ref}`constraint-instance-type`      | &#10003;  <br> Valid values: See cloud provider. <br> Default value: `m3.medium`.                                                                                 |
-| - {ref}`constraint-mem`                | &#10003;                                                                                                                                                          |
-| - {ref}`constraint-root-disk`          | &#10003;                                                                                                                                                          |
-| - {ref}`constraint-root-disk-source`   | &#10003;                                                                                                                                                          |
-| - {ref}`constraint-spaces`             | &#10003;                                                                                                                                                          |
-| - {ref}`constraint-tags`               | &#10005;                                                                                                                                                          |
-| - {ref}`constraint-virt-type`          | &#10005;                                                                                                                                                          |
-| - {ref}`constraint-zones`              | &#10003;                                                                                                                                                          |
-
+| {ref}`CONSTRAINT <constraint>`         |                                                     |
+|----------------------------------------|-----------------------------------------------------|
+| conflicting:                           | `instance-type` vs. `[cores, cpu-power, mem]` |
+| supported?                             |                                                     |
+| - {ref}`constraint-allocate-public-ip` | &#10003;                                            |
+| - {ref}`constraint-arch`               | &#10003;                                            |
+| - {ref}`constraint-cores`              | &#10003;                                            |
+| - {ref}`constraint-cpu-power`          | &#10003;                                            |
+| - {ref}`constraint-image-id`           | &#10003; <br> An AMI.                               |
+| - {ref}`constraint-instance-role`      | &#10005; <br> Value: `auto` or an [instance profile](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html) name.|
+| - {ref}`constraint-instance-type`      | &#10003; <br> Valid values: See cloud provider. <br> Default value: `m3.medium`.|
```

Comment:
Just noticed this now, we don't have a default of "m3.medium". It is calculated and could change.